### ISSUE TITLE
Predicting on training datasets

### ISSUE THREAD

**Issue by BordoBeagle** [ourownstory/neural_prophet#508]
Hi, 

documentation shows that it should be possible to use predict of training dataframe. However, when I am trying to do it I get and error about missing 't' column.

**Comment by christopherhan920** [comment]
I'm getting this error as well. the model declared as below.

m = NeuralProphet(
    growth="linear",  
    changepoints=None, 
    n_changepoints=30,
    changepoints_range=0.9,
    yearly_seasonality="auto",
    weekly_seasonality="auto",
    daily_seasonality="auto",
    normalize="auto",
    seasonality_mode="multiplicative",
    epochs=100,
    n_lags=0
)

![image](https://user-images.githubusercontent.com/97553144/149007311-7353697d-1df1-49ec-a39c-c6ad402b9f09.png)

**Comment by ourownstory** [comment]
Hi @BordoBeagle and @christopherhan920 
Please excuse the late reply.
Is this still and issue with 0.3.2 or 0.3.3?

If so, please reopen the issue and provide a minimal set of code to reproduce the issue.

Thank you
